tutorials/dask.md

```diff
@@ -0,0 +1,149 @@
+# using dask and napari to process & view large datasets
+
+Often in microscopy, multidimensional data is acquired and written to disk in many small files, each of which contain a subset of one or more dimensions from the complete dataset.  For example, in a 5-dimensional experiment (e.g. 3D z-stacks aquired for multiple channels at various moments over time), each file on disk might be a 3D TIFF stack from a single channel at a single timepoint.  Data may also be stored in some proprietary format.  As the size of the dataset grows, viewing arbitrary slices (in time, channel, z) of these datasets can become cumbersome.
+
+Chunked file formats exist (such as [hdf5](https://support.hdfgroup.org/HDF5/) and [zarr](https://zarr.readthedocs.io/en/stable/)) that store data in a way that makes it easier to retrieve arbitrary subsets of the dataset, but they require either data duplication, or "committing" to a new file standard.
+
+> **Note**: This tutorial is not meant to promote a folder of TIFFs as a "good way" to store large datasets on disk; but it is an undoubtedly common scenario in microscopy.  Chunked formats such as `hdf5` or `zarr` are superior in many ways, but they do require the user to either duplicate their data, or go "all in" and delete the original data after conversion.  And while `napari` can easily handle something like a `zarr` store, it can be a bit more limiting inasmuch as it requires programs that capable of viewing it (i.e. you can't currently simply drag it into Fiji ...)
```

Comment:
Basically there was a missing "are". The GitHub diff doesn't make that abundantly clear though.

```suggestion
> **Note**: This tutorial is not meant to promote a folder of TIFFs as a "good way" to store large datasets on disk; but it is an undoubtedly common scenario in microscopy.  Chunked formats such as `hdf5` or `zarr` are superior in many ways, but they do require the user to either duplicate their data, or go "all in" and delete the original data after conversion.  And while `napari` can easily handle something like a `zarr` store, it can be a bit more limiting inasmuch as it requires programs that are capable of viewing it (i.e. you can't currently simply drag it into Fiji ...)
```